multicluster/test/e2e/framework.go

```diff
@@ -62,6 +62,7 @@ const (
 
 	nginxImage   = "nginx:latest"
 	agnhostImage = "agnhost:2.26"
+	nginxPodName = "test-nginx-pod"
```

Comment:
any reason why we do not use randName for ngnix pod name generation anymore?

Reply:
Yeah, so now we put the setup and teardown of pod into two functions in service_test.go: setupTestResources and tearDownTestResources. If we use random name, how does the teardown know the pod names? Do you mean we should let setupTestResources return the pod names, and pass the names in tearDownTestResources?

Reply:
Is there a possibility multiple concurrent tests will try and create a pod with this name in the same namespace?